Sources/Shared/Classes/waterwheel.swift

```diff
@@ -335,7 +343,20 @@ public func sendRequestWithUrl(_ urlString: String, method: Alamofire.HTTPMethod
 
  */
 public func get(_ requestPath: String, params: paramType, completionHandler: completion?) {
-    sendRequest(requestPath, method: .get, params: nil) { (success, response, json, error) in
+
+    var ultimateRequestPath = requestPath
+
+    if params != nil {
+        let getParams = params!.map( { key, value -> String in
+            return "\(key)=\(value)"
+        }).joined(separator: "&")
+
+        if getParams != "" {
+            ultimateRequestPath = requestPath + "?" + getParams
+        }
+    }
+
+    sendRequest(ultimateRequestPath, method: .get, params: nil) { (success, response, json, error) in
```

Comment:
We should really be sending the params along with the Alamofire Request. 

_format=json should be added as object into that array. If that makes sense.

Reply:
I'm not very familiar with Almoafire. Will it take the params arg and properly url encode the param dict?

Reply:
Yeah, notice the params parameter is nil. Thats a mistake and it should pass the params along. 

And in sendRequest, we should append the format, instead of adding it to the path. Alamofire will handle adding it. if it that makes sense. 

Im happy to accept this though if you dont have time to mess with it.

I can fix it later this this week.

Reply:
It'd be great if we could merge this (I don't want to accidentally overwrite it with a pod install or something) and I'll likely get curious and play with it myself and issue another PR if that works for you?

At least this solution works "for now"...?